src/plugins/graph/models/graph-layer-model.ts

```diff
@@ -153,3 +157,7 @@ export const GraphLayerModel = types
 
 export interface IGraphLayerModel extends Instance<typeof GraphLayerModel> { }
 export interface IGraphLayerModelSnapshot extends SnapshotIn<typeof GraphLayerModel> {}
+
+export const GraphLayerContext = createContext<IGraphLayerModel>({} as IGraphLayerModel);
+
+export const useGraphLayerContext = () => useContext(GraphLayerContext);
```

Comment:
I would move this to its own file to avoid bringing a React dependency into the model.

Reply:
Done. I also moved GraphModelContext since it was coded the same way.